src/components/text-input/text-input.hbs

```diff
@@ -42,6 +42,9 @@
 
 <div {{#if password}} data-text-input {{/if}} class="bx--form-item bx--text-input-wrapper">
   <label for="text-input-5" class="bx--label">Text Input label</label>
+  <div class="bx--form__helper-text">
```

Comment:
Wanted to double-check if the markup change works for stable mode (or stable mode was like this before).

Reply:
Double checked and seems ok 👍